src/clients/sap_ha.rb

```diff
@@ -54,6 +55,10 @@ class SAPHAClass < Client
 
     def initialize
       log.warn "--- called #{self.class}.#{__callee__}: CLI arguments are #{WFM.Args} ---"
+      #Take care that corosync is enabled and running
+      corosync = Yast2::Systemd::Service.find('corosync')
+      corosync.enable if !corosync.enabled?
```

Comment:
Actually, I would say it is safe to call `#enable` and `#start` even if the system is enabled and running (systemd will not do any change). However, it might be a good idea to check whether the `corosync` service actually exists.

```ruby
corosync = Yast2::Systemd::Service.find('corosync')
if corosync
  corosync.enable
  corosync.start
else
  log.warn "corosync service was not found"
end
```

If for some reason the package is not installed, or the service is renamed, or... YaST will crash. But it is just a suggestion.